tests/lib/EventListener/ContentProxyCreateDraftListenerTest.php

```diff
@@ -142,6 +146,10 @@ public function testCreateContentOnTheFlyAutosaveEnabled(): void
         $eventDispatcher->dispatch($createEvent);
 
         $this->assertEquals(new RedirectResponse('redirect_on_the_fly_test_url'), $createEvent->getResponse());
+        self::assertInstanceOf(
+            Content::class,
+            $createEvent->getOptions()->get(ContentProxyCreateEvent::OPTION_CONTENT_DRAFT)
```

Comment:
For the purposes of the test however, you should not rely on the constant. Constant helps to notice the connection between classes and gives meaning to a simple string, but what we care in the end is that an array with specific key exists. 

This will prevent or significantly reduce the chances of someone creating a BC break by changing the constant.

Reply:
Fixed: 6fef5cb